act-rules-format.md

```diff
@@ -170,42 +170,85 @@ Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em>
+
+An ACT Rule <em class="rfc2119">may</em> show that the outcomes of the rule have related or secondary [=accessibility requirements=] for that test subject. The secondary accessibility requirements may not be required for conformance to the [=accessibility requirements document=]. When one or more of the outcomes for a test target is `failed`, the secondary accessibility requirement could be <dfn>not satisfied</dfn> for the test subject or <dfn>further testing is needed</dfn> . When all of the outcomes are `passed` or `inapplicable`, the secondary accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>. The rule must explain the secondary relationship of the accessibility requirement to the rule. 
+
+<aside class=example>
+  <header>Example accessibility requirements mapping for a rule that tests if text has minimum contrast:</header>
+  <blockquote><ul>
     <li>
-      [Success criterion 4.1.2: Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)
+      [Success criterion 1.4.3 Contrast Minimum (AA)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
       <ul>
         <li>**Required for conformance** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
         <li>Outcome mapping:<ul>
           <li>Any `failed` outcomes: not satisfied</li>
-          <li>All `passed` outcomes: further testing is needed</li>
+          <li>All `passed` outcomes: satisfied</li>
+          <li>An `inapplicable` outcome: further testing is needed</li>
+        </ul></li>
+      </ul>
+    </li>
+    <li>
+      [Success Criterion 1.4.6 Contrast (Enhanced) (Level AAA)](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
+      <ul>
+        <li>**Secondary** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
+        <li>Outcome mapping:<ul>
+          <li>Any `failed` outcomes: not satisfied</li>
+          <li>All `passed` outcomes: further testing is needed; SC 1.4.6 requires higher contrast ratios than SC 1.4.3</li>
+          <li>An `inapplicable` outcome: further testing is needed</li>
+        </ul></li>
+      </ul>
+    </li>
+  </ul></blockquote>
+</aside>
+
+<aside class=example>
+  <header>Example accessibility requirements mapping for a rule that tests if a focusable element has no keyboard trap via standard navigation:</header>
+  <blockquote><ul>
+    <li>
+      [Success Criterion 2.1.2: No Keyboard Trap](https://www.w3.org/TR/WCAG21/#no-keyboard-trap)
+      <ul>
+        <li>**Secondary** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
+        <li>Outcome mapping:<ul>
+          <li>Any `failed` outcomes: further testing is needed; success criterion permits use of non-standard navigation to exit a keyboard trap</li>
+          <li>All `passed` outcomes: satisfied</li>
           <li>An `inapplicable` outcome: further testing is needed</li>
         </ul></li>
       </ul>
     </li>
   </ul></blockquote>
 </aside>
 
+### Mapping for Atomic Rules ### {mapping-for-atomic-rules}
```

Comment:
I think this may need to be flipped. Atomic rules do not have a relationship with composite rules. I.e. you could in theory add a composite rule without updating the atomic rule. I think this requirement should be flipped, in that a composite rule must not include accessibility requirements unless they are part of all its atomic rules. I.e. the compisite rule has the exact same accessibility requirements as its atomic rules. The main difference being that what is secondary and what is not will differ.

Reply:
From how I am reading it, lines 248 and 251 are saying the same thing: an atomic rule cannot map to something that the composite rule does not map to. 

In 248 I read it as the atomic rule must map to a (possibly improper) subset of the composite rule's mapping. And in 251, I read it as the composite rule must map to the union of all of the mappings (conformance and secondary) present in the atomic rules. 

So I'm not sure it needs to be flipped, but some clarification might be helpful.

Reply:
What about this (removing reference to composite):

Conformance requirements (that are not satisfied) and secondary requirements (that may or may not be satisfied) must be included in the Accessibility Requirements of atomic rules.